kartothek/core/cube/constants.py

```diff
@@ -0,0 +1,48 @@
+"""
+Common constants for klee.
+"""
+from kartothek.serialization import ParquetSerializer
+
+__all__ = (
+    "KLEE_DF_SERIALIZER",
```

Comment:
should probably change these to smth like `KTK_CUBE_*`

Reply:
done.